server/chat-commands.js

```diff
@@ -898,8 +898,8 @@ const commands = {
 
 	'!groupchatuptime': true,
 	groupchatuptime(target, room, user) {
+		if (!room || !room.uptime) return this.errorReply("Can only be used in a groupchat.");
```

Comment:
This isn't a complete sentence; "This command can only be used in a groupchat." sounds better

Reply:
It doesn't need to be a complete sentence.